engine/glfw/lib/cocoa/cocoa_window.m

```diff
@@ -127,9 +127,17 @@ - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication *)sende
     return NSTerminateCancel;
 }
 
-- (void)applicationDidFinishLaunching:(NSNotification *)notification
-{
-    [[NSRunningApplication currentApplication] activateWithOptions:(NSApplicationActivateAllWindows | NSApplicationActivateIgnoringOtherApps)];
+- (void)applicationDidUpdate:(NSNotification *)notification
+{
+    // Wait for the first update to make window active
+    // It helps to avoid issue when the window opens inactive
+    // https://github.com/defold/defold/issues/6709
+    if( !_glfwLibrary.Unbundled ) {
+        ProcessSerialNumber psn = { 0, kCurrentProcess };
+        TransformProcessType( &psn, kProcessTransformToForegroundApplication );
+        [[NSApplication sharedApplication] activateIgnoringOtherApps: YES];
```

Comment:
Recommended function instead of `SetFrontProcess()` even though it's deprecated as well 🤷‍♂️